### PR TITLE
Support redeployable graphql server

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

**Comment by eliperelman** [comment]
Good idea!

**Comment by eliperelman** [comment]
Done.

File: README.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ PULSE_USERNAME="<insert username here>"
 
 # Password for connecting to pulse for subscriptions:
 PULSE_PASSWORD="<insert password here>"
+
+# Hostname for connecting to pulse for subscriptions:
+PULSE_HOSTNAME="pulse.mozilla.org"
+
+# VHost for connecting to pulse for subscriptions:
+PULSE_VHOST="/"
```
Comment (helfi92) [inline README.md]: Should we add `TASKCLUSTER_ROOT_URL` as well?

Comment (eliperelman) [reply README.md]: Yes, fail

**Comment by eliperelman** [comment]
OK, after some fun times updating libs, this should now be fixed. I tested out queries and subscriptions and it worked. Give it a shot and lemme know.

**Pull request merged**